src/GraphQL/Utilities/LinkConfiguration.cs

```diff
@@ -0,0 +1,298 @@
+using System.Collections;
+using GraphQL.Types;
+using GraphQL.Types.Scalars;
+
+namespace GraphQL.Utilities;
+
+/// <summary>
+/// Configuration for the @link directive, allowing customization of the linked schema.
+/// </summary>
+public sealed class LinkConfiguration
+{
+    internal const string LINK_URL = "https://specs.apollo.dev/link/v1.0";
+    internal const string METADATA_KEY = "__Link_Configuration";
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="LinkConfiguration"/> class with the specified URL.
+    /// </summary>
+    public LinkConfiguration(string url)
+    {
+        Url = url;
+    }
+
+    /// <summary>
+    /// The URL of the linked schema.
+    /// </summary>
+    public string Url { get; set; }
```

Comment:
If the URL is required via constructor, then this property probably should be get-only.

Reply:
Ok I’ll change that.